### DISCUSSION TITLE
What is "World" tracking?

### DISCUSSION THREAD

**Discussion by ReallyRilee** [SavedInstances/SavedInstances#888] [Q&A] [answered]
I just started using this addon to track all my character's weekly chores, and I can't figure out what "world" is tracking. It seems to be wildly different on my 2 characters. One has 1|1 and the other has 8|8|1. Any help would be appreciated! 
![image](https://github.com/user-attachments/assets/332fdc99-9698-4121-ac68-256882867b91)

**Comment by LiangYuxuan** [comment, marked as answer]
"World" is the Great Vault's World row, mainly delves.
